app/notification.go

```diff
@@ -333,8 +333,8 @@ func sendNotificationEmail(post *model.Post, user *model.User, channel *model.Ch
 	if *utils.Cfg.EmailSettings.EnableEmailBatching {
 		var sendBatched bool
 		if result := <-Srv.Store.Preference().Get(user.Id, model.PREFERENCE_CATEGORY_NOTIFICATIONS, model.PREFERENCE_NAME_EMAIL_INTERVAL); result.Err != nil {
-			// if the call fails, assume it hasn't been set and don't batch notifications for this user
-			sendBatched = false
+			// if the call fails, assume that the interval has not been explicitly set and batch the notifications
+			sendBatched = true
```

Comment:
This commit fixes the problem because the preference store interface is implemented by `sql_preference_store.go`, which uses `gorp` to connect to the database. That implementation calls the [SelectOne](https://godoc.org/github.com/go-gorp/gorp#SelectOne) method on the query result set, which raises an error if there are zero results, indicating in this case that the user has not explicitly set this preference value.